### PR TITLE
Fixed tutorial code sample

### PR DESCRIPTION
**Pull request by blue32a**
## Description

When the current code sample is executed, the API response does not return the intended data. (#1631)
This PR fixes that problem.

### CODE REVIEW

File: src/content/graphql-js/Tutorial-Authentication.md [hunk 1]
```diff
@@ -42,7 +42,9 @@ app.all(
   createHandler({
     schema: schema,
     rootValue: root,
-    graphiql: true,
+    context: (req) => ({
+      ip: req.raw.ip,
```
Comment (benjie) [inline src/content/graphql-js/Tutorial-Authentication.md]: I don't see `req.raw` in Express' documentation; @enisdenjo does graphql-http wrap the request object when passed to context callback?

Comment (enisdenjo) [reply src/content/graphql-js/Tutorial-Authentication.md]: graphql-http handler is agnostic to the underlying implementation and req.raw is the request of the implementation. [req is actually a graphql-http request](https://github.com/graphql/graphql-http/blob/main/docs/interfaces/handler.Request.md).

**Review by enisdenjo — APPROVED** [review]

File: src/content/graphql-js/Tutorial-Authentication.md (hunk 1 above)
Comment (benjie) [reply src/content/graphql-js/Tutorial-Authentication.md]: So: "yes" :grin:

**Review by benjie — APPROVED** [review]

**Comment by Urigo** [comment]
@blue32a can you try to resolve the new conflicts?

**Comment by blue32a** [comment]
@Urigo conflicts resolved.

**Comment by Urigo** [comment]
@blue32a thank you!

**Pull request merged**